web/webplatform/fetch.sh

```diff
@@ -4,28 +4,45 @@ set -euo pipefail
 
 cd "$(dirname "$0")"
 
-VERSION="$(cat VERSION)"
+VERSION="$(cut -w -f 1 < VERSION)"
+SHA="$(cut -w -f 2 < VERSION)"
 FILENAME="autokitteh-web-v${VERSION}.zip"
 
-echo "want version ${VERSION}"
+echo "want version ${VERSION}."
+
+checksum() {
+    echo "verifying checksum..."
+
+    sha=$(shasum -a 256 "$1" | cut -d ' ' -f 1)
+
+    if [[ "${sha}" != "${SHA}" ]]; then
```

Comment:
This is hard to understand, please use clearer and unique var names like "expected" and "actual" instead of "SHA" and "sha".

And please use a single convention for var names: either "FOO" or "foo".

Reply:
fixed

Reply:
oh, re convention - "sha" is a "local" var, others are inputs/globals/whatever